.ci/Jenkinsfile

```diff
@@ -188,7 +217,7 @@ def runReleaseWithKeychain() {
     error("runReleaseWithKeychain: macos secret could not be accessed correctly")
   }
   def token = getVaultSecret.getVaultToken(vault.vault_addr, vault.vault_role_id, vault.vault_secret_id)
-  def apple = getVaultSecret.getVaultSecretObject(vault.vault_addr, 'secret/release/apple', token).data
+  def apple = getVaultSecret.getVaultSecretObject(vault.vault_addr, 'secret/release/apple-2023', token).data
```

Comment:
Fixes the new vault secret. Unrelated to the nature of this particular PR, otherwise it won't work the releases.